lib/maplibre-compose-expressions/src/commonMain/kotlin/dev/sargunv/maplibrecompose/expression/value/CirclePitchAlignment.kt

```diff
@@ -0,0 +1,13 @@
+package dev.sargunv.maplibrecompose.expression.value
+
+import dev.sargunv.maplibrecompose.expression.ast.StringLiteral
+
+/** Orientation of circles when the map is pitched. */
+public enum class CirclePitchAlignment(override val literal: StringLiteral) :
```

Comment:
You've put the expression stuff into an own dependency now. But `CirclePitchAlignment` and all the others are not really directly related to expressions but rather to layer properties, aren't they?

Reply:
they're implementations of a `sealed interface EnumValue` so they need to be in the same module. And I see them as specializations of String, similar to how DpValue, TextUnitValue, etc are specializations of NumberValue.